app/controllers/admin/users_controller.rb

```diff
@@ -49,6 +49,16 @@ def user_status
       redirect_to "/admin/users/#{@user.id}/edit"
     end
 
+    def export_data
+      user = User.find(params[:id])
+      using StringToBool
```

Comment:
This line is blowing up in my local env; I'm getting the error:
```
undefined method `using' for #<Admin::UsersController:0x00007fa43cb00990>
```

This is happening for both `Export to Admin` and `Export to User` buttons.

Reply:
Great catch @msarit. I don't think `using` can be inside a method call. According to https://docs.ruby-lang.org/en/2.7.0/Module.html#method-i-using it can be called inside a module or a class definition.

In this case I think it should be:

```ruby
module Admin
  class UsersController < Admin::ApplicationController
    using StringToBool

    layout "admin"

    # ...
  end
end
```

Reply:
Yes, that's correct. My example was a copy/paste from an IRb session, where you're implicitly in the scope of the `main` module, so `using` just works.

Reply:
Ahh I see. Thanks for catching this!